src/App.js

```diff
@@ -1,61 +1,73 @@
 import React from 'react';
+import InputTodo from './Components/InputTodo';
+import TodoList from './Components/TodoList';
+import Footer from './Components/Footer';
 
-function App() {
-  return (
-    <section className="todoapp">
-      <header className="header">
-        <h1>todos App</h1>
+class App extends React.Component {
+  state = {
+    todo: [],
```

Comment:
todo? один? а присвоєно масив:(
```suggestion
    todos: [],
```